cohydra/node/base.py

```diff
@@ -33,8 +35,17 @@ def __init__(self, name):
         self.ns3_node = network.Node()
         core.Names.Add(self.name, self.ns3_node)
 
-        self.set_position(random.randint(10, 100), random.randint(10, 100))
-        #: The color of the node used in NetAnim.
+        #: The position of the node (used by wifi models and visualisation)
+        self.position = (0, 0, 0)
+        # Every change at the position will be forwarded to the visualisation
+        aexpr(lambda: self.position, globals(), locals())\
```

Comment:
Since we are in python you can always call `object.position = (1, 2, 3)` without using the setter. This makes sure, that the position in the model is always the same as in the visualization.
I would prefer this active expressions way but I think, it would be better, if this would be moved to the visualization class. Than the model hasn't any visualization code inside.